docs/sources/s3/class-source-s3-parser.md

```diff
@@ -0,0 +1,9 @@
+## Sources.S3.Parser
+Parser is the class that will translate strings (starting with yamal) into properties (json) between storage and services
```

Comment:
In this case, no YAML will be involved. This parser will just be translating data pulled from the aws-sdk into our storage layer's json schema.